unlock-app/src/pages/dashboard.js

```diff
@@ -35,4 +41,4 @@ const mapStateToProps = state => {
   }
 }
 
-export default connect(mapStateToProps)(Dashboard)
+export default withConfig(connect(mapStateToProps)(Dashboard))
```

Comment:
Why do you need to wrap this in `withConfig` since this code does not seem to be using the config?

Reply:
If I understood this right, dashboard is depending on connecting the Ethereum network. I am wrapping every page that uses the `network` with `withConfig` since it has the "MissingProvider" and "WrongNetwork" error logic, worth adding loading state there too if it's trying to connect to the network.

Reply:
Yes you are right, the dashboard *needs* the network stuff... I see your point now. It's kind of weird that we wrap this but not actually use anything here. I'll think more about it.